react/pages/about/PricingPage/index.js

```diff
@@ -0,0 +1,190 @@
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+
+import constants from 'react/styles/constants';
+import Box from 'react/components/UI/Box';
+import Text from 'react/components/UI/Text';
+
+import { Headline, Subheadline, Description } from 'react/pages/about/components/Text';
+import CenterBox from 'react/pages/about/components/CenterBox';
+import PricingTable from 'react/pages/about/PricingPage/components/PricingTable';
+
+const Container = styled.div`
+  margin-top: 4em;
```

Comment:
There's rarely an instance where you need to be inputting values manually, if so it's usually for a very good reason.